client/dashboard/profile-wizard/steps/industry.js

```diff
@@ -52,13 +52,19 @@ class Industry extends Component {
 		const selectedIndustriesList = this.state.selected.map(
 			( industry ) => industry.slug
 		);
-		const industriesWithDetail = filter( this.state.selected, ( value ) => {
-			return typeof value.detail !== 'undefined';
+
+		// With this 'forEach' we'll record all the 'industries_with_detail' available.
+		// For now it's only being used for the input 'Other'.
+		forEach( this.state.selected, ( value ) => {
+			if ( typeof value.detail !== 'undefined' ) {
+				recordEvent( 'storeprofiler_store_industry_continue', {
+					industries_with_detail: value.detail,
+				} );
```

Comment:
I don't think we should record duplicate events just to capture the "other" fields.

Why not turn the `this.state.selected` values into a comma separated list of string `.detail` values?

Reply:
Sounds good! I'll do it.
Thank you :slightly_smiling_face:

Reply:
Solved in commit [3c0a6ae](https://github.com/woocommerce/woocommerce-admin/pull/3929/commits/3c0a6ae09b190f613ac85e718fdc9e4eca30a03d).
The selected industries details are converted to a string (comma-separated list) to send to the `recordEvent` method